Makefile

```diff
@@ -11,7 +11,7 @@
 #   Stable releases:   "1.0.0"
 #   Pre-releases:      "1.0.0-alpha.1", "1.0.0-beta.2", "1.0.0-rc.3"
 #   Master/dev branch: "1.0.0-dev"
-VERSION=12.0.0-dev
+VERSION=12.0.0
```

Comment:
Is this intended?

Reply:
Yeah we should rollback this change until we cut `branch/v12`.

Reply:
Fixed - thanks for catching it